src/content/en/tools/chrome-devtools/issues/index.md

```diff
@@ -77,7 +76,11 @@ title="SameSite cookie tests">samesite-sandbox.glitch.me</a>) with issues to fix
 
 ## View items in the Issues tab {: #view-issues }
 
-1. Click an item in the **Issues** tab.
+The **Issues** tab presents warnings from the browser in a structured, aggregated, and 
+actionable way. 
+
+1. Click an item in the **Issues** tab to get guidance on how to fix the issue and view 
```

Comment:
Maybe 'how to fix issues and find affected resources'.

Reply:
Done, thanks for reviewing it! I guess I need your LGTM again